packages/terra-hyperlink/src/Hyperlink.module.scss

```diff
@@ -8,11 +8,14 @@
     background-color: transparent; // Remove the gray background on active links in IE 10
     color: var(--terra-hyperlink-color, #006fc3);
     cursor: pointer;
-    display: inline-block; // prevents variant icon from breaking onto its own line
     outline: none;
     text-decoration: var(--terra-hyperlink-text-decoration, underline);
     touch-action: manipulation; // Enable fast tap interaction in webkit browsers; see https://webkit.org/blog/5610/more-responsive-tapping-on-ios/
     vertical-align: baseline;
+    display: inline-flex;
```

Comment:
Why are we changing the style from inline-block to inline-flex?  I would not expect this change to be necessary.

Reply:
Since we're needing to have 2 spans with 1 being an ellipses but another being on the same level, we'll need to go with this.

Reply:
Also inline-block wasn't allowing us to properly truncate without the hyperlink "clickable" area extending past the "visual" area of the hyperlink. Using inline-flex, we were able to properly have the "text" portion of hyperlink resize itself so it would truncate without issue while allowing the icon to always render at full size.